src/blocks/9112Michael/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Hacktoberfest 2022 Project">
+    <link rel="stylesheet" href="style.css">
+    <title>Hactoberfest 2022 Webblocks Contribution</title>
+</head>
+<body>
+    <header>
+        <h1>Happy Hacktoberfest</h1>
+        <p>Edit contents of the inner box!</p>
+      </header>
+      <main class="outer">
+        <div class="inner" contenteditable="true">Happy Hacktoberfest!</div>
+      </main>
+    <script src="script.js"></script>
```

Comment:
You could remove the `script` tag and the JavaScript file if you don't use it.

Reply:
I have removed the script and file from this project.